ios/RCTTWVideoModule.m

```diff
@@ -239,40 +246,136 @@ - (void)room:(TVIRoom *)room didFailToConnectWithError:(nonnull NSError *)error{
+    
+    // We are subscribed to the remote Participant's audio Track. We will start receiving the
+    // remote Participant's video frames now.
+    
+    [self logMessage:[NSString stringWithFormat:@"Subscribed to video track for Participant %@", participant.identity]];
+    
+      [self sendEventWithName:participantAddedVideoTrack body:@{ @"participant": [participant toJSON], @"track": [videoTrack toJSON] }];
+}
+
+- (void)unsubscribedFromVideoTrack:(TVIRemoteVideoTrack *)videoTrack
+                       publication:(TVIRemoteVideoTrackPublication *)publication
+                    forParticipant:(TVIRemoteParticipant *)participant {
+    
+    // We are unsubscribed from the remote Participant's video Track. We will no longer receive the
+    // remote Participant's video.
+    
+    [self logMessage:[NSString stringWithFormat:@"Unsubscribed from video track for Participant %@", participant.identity]];
+    
+    [self sendEventWithName:participantRemovedVideoTrack body:@{ @"participant": [participant toJSON], @"track": [videoTrack toJSON] }];
+}
+
+- (void)subscribedToAudioTrack:(TVIRemoteAudioTrack *)audioTrack
+                   publication:(TVIRemoteAudioTrackPublication *)publication
+                forParticipant:(TVIRemoteParticipant *)participant {
+    
+    // We are subscribed to the remote Participant's audio Track. We will start receiving the
+    // remote Participant's audio now.
+    
+    [self logMessage:[NSString stringWithFormat:@"Subscribed to audio track for Participant %@", participant.identity]];
+    [self sendEventWithName:participantAddedAudioTrack body:@{ @"participant": [participant toJSON], @"track": [audioTrack toJSON] }];
+}
+
+- (void)unsubscribedFromAudioTrack:(TVIRemoteAudioTrack *)audioTrack
+                       publication:(TVIRemoteAudioTrackPublication *)publication
+                    forParticipant:(TVIRemoteParticipant *)participant {
+    
+    // We are unsubscribed from the remote Participant's audio Track. We will no longer receive the
+    // remote Participant's audio.
+    
+    [self logMessage:[NSString stringWithFormat:@"Unsubscribed from audio track for Participant %@", participant.identity]];
+    [self sendEventWithName:participantRemovedAudioTrack body:@{ @"participant": [participant toJSON], @"track": [audioTrack toJSON] }];
+}
 
-- (void)participant:(TVIParticipant *)participant addedVideoTrack:(TVIVideoTrack *)videoTrack {
-  [self sendEventWithName:participantAddedVideoTrack body:@{ @"participant": [participant toJSON], @"track": [videoTrack toJSON] }];
+- (void)remoteParticipant:(TVIRemoteParticipant *)participant
+        enabledVideoTrack:(TVIRemoteVideoTrackPublication *)publication {
+    [self logMessage:[NSString stringWithFormat:@"Participant %@ enabled video track.", participant.identity]];
+//TBA      [self sendEventWithName:participantEnabledTrack body:@{ @"participant": [participant toJSON], @"track": [publication toJSON] }];
```

Comment:
I think that we can add this events without issues. We only need to add them in JS side for documentation